stack.yaml

```diff
@@ -3,6 +3,7 @@ packages:
 - .
 - lib/bech32
 - lib/core
+- lib/core/test/integration
```

Comment:
I guess this makes sense, but I'm curious if you considered not having `core` in the path/name: e.g `lib/integration`?